### DISCUSSION TITLE
In the filters in the stats screen, add the ability to EXCLUDE tags

### DISCUSSION THREAD

**Discussion by victor-marino** [enrique-lozano/Monekin#265] [Ideas / New features]
This is a great feature that I've seen in the (now unsupported) Ivy Wallet app, and that I'd love to see implemented here.

Like title says, it'd be great if the filters also allowed us to exclude specific tags from the stats.

Here's what it looks like in Ivy Wallet:
![negative_filters](https://github.com/user-attachments/assets/bc0e7f88-8eb3-450b-b922-08f20190f149)

For instance, I may want to find out how much I've spent on _Food_ (category), while _Traveling_ (tag) but excluding _Work_ meals (tag).

What do you think?

**Comment by enrique-lozano** [comment]
Hmmm I don't see it, this is the same as select all the tags that you want except the ones you want to exclude right? You also have a "Select/Unselect All" button to facilitate this process

**Reply by victor-marino** [reply]
Hmm not exactly.

Imagine this scenario:
* Transaction A tags: `Work`, `Travel`
* Transaction B tags: `Travel`

Currently, if I want to filter items in the `Food` category that include the `Travel`tag, the filter will include both A and B. Even if I don't select `Work`, transaction A will be included, because it does contain the `Travel` tag.

In this case, it's possible that your work expenses are eventually being reimbursed by your employer, and so you want to log them and classify them correctly, but have the option to exclude them when creating reports.

To provide another example, imagine you're having certain food/clothing/purchase transactions that have been associated to a very particular, one-off event (say, your car broke down in the middle of a trip and you have to purchase extra stuff). Perhaps at the end of the year you'd like to see your expenses but exclude the ones derived from this particular event.

Negative filters are very powerful for this kind of scenarios.

**Reply by enrique-lozano** [reply]
Oh, I see, you're absolutely right! Thank you very much for the explanation. 

I'm adding it as a new issue to do because I find it a quite interesting feature. We can just add a new card in the filter sheet below the current "Tags" card.

**Comment by enrique-lozano** [comment]
Issue created in #279 with all the basic requirements, in case anyone wants to work on it 😊
